### ISSUE TITLE
add number to new file if already exists

### ISSUE THREAD

**Issue by imeed166** [vanadium23/obsidian-advanced-new-file#29]
Hey, little feature request for adding a number to the new file is already taken, instead of displaying an error.

**Comment by iamdannywyatt** [comment]
> Hey, little feature request for adding a number to the new file is already taken, instead of displaying an error.

I agree with this. Incrementing the file name would be a great feature
